README.md

```diff
@@ -0,0 +1,125 @@
+# Gruntwork Installer
+
+[Gruntwork Script Modules](https://github.com/gruntwork-io/script-modules) is a private repo that contains scripts and
+applications developed by [Gruntwork](http://www.gruntwork.io) for common infrastructure tasks such as setting up
+continuous integration, monitoring, log aggregation, and SSH access. This repo contains provides a script called
```

Comment:
Fixed as part of https://github.com/gruntwork-io/gruntwork-installer/commit/fd4e92a8b3bf504e6ec47e45df4ca592b21fedb9